docs/en_US/preferences.rst

```diff
@@ -497,9 +497,8 @@ preferences for copied data.
 * Specify the maximum width of the column in pixels when 'Columns sized by' is
   set to *Column data*. If 'Columns sized by' is set to *Column name* then this
   setting won't have any effect.
-* Specify the number of records to fetch in one batch in query tool when
-  query result set is large. Changing this value will override
-  ON_DEMAND_ROW_COUNT setting from config file.
+* Specify the number of records to fetch in one batch. Changing this value will
+  override DATA_RESULT_PAGE_SIZE setting from config file.
```

Comment:
Rename this to **RECORDS_PER_PAGE** or **ROWS_PER_PAGE** instead of DATA_RESULT_PAGE_SIZE.